script.js

```diff
@@ -189,33 +189,42 @@ var conveyorBelt = document.createElement('div');
 
 
 var containers = {
-    bin1: { element: document.createElement('img'), x: 100, y: 600 },
-    bin2: { element: document.createElement('img'), x: 300, y: 600 },
-    bin3: { element: document.createElement('img'), x: 600, y: 600 },
-    bin4: { element: document.createElement('img'), x: 1000, y:600 }
-}
+    bin1: { element: document.createElement('img'), x: 100, y: 600, itemIndexInBin: [] },
+    bin2: { element: document.createElement('img'), x: 300, y: 600, itemIndexInBin: [] },
+    bin3: { element: document.createElement('img'), x: 600, y: 600, itemIndexInBin: [] },
+    bin4: { element: document.createElement('img'), x: 1000, y:600, itemIndexInBin: [] }
+};
 
-// var playerHead = document.createElement('div');
-// var playerBody = document.createElement('div');
-// var spine = document.createElement('div');
-// var leftHand = document.createElement('div');
-// var rightHand = document.createElement('div');
 
 function paintBins() {
 
     for (var i in containers) {
         containers[i].element.style.left = containers[i].x;
         containers[i].element.style.top = containers[i].y;
         containers[i].element.className = `${i}`;
+        containers[i].element.id = `{i}`;
```

Comment:
Think you're missing a $ here - this will just print the literal string `"{i}"`.